nginx-log-metrics/README.md

```diff
@@ -0,0 +1,66 @@
+# Nginx log metrics demo
+
+## Quick Start
+
+Build and start with proxy:
+
+```shell
+docker compose build --build-arg https_proxy=http://172.17.0.1:7890
```

Comment:
Do we need to mention proxy here?

Reply:
I make something similiar at https://github.com/GreptimeTeam/demo-scene/tree/main/influxdb-lineprotocol#step-2a-write-data-via-line-protocol for proxy. But I believe it's better to have a dedicated block or we just outlink.

Typical network issue varies and has different solution for different people. This command may not work for some of them and it's out of the scope of this demo.